### PR TITLE
Add SIG Architecture Conformance spotlight interview

### PR DESCRIPTION
**Pull request by fsmunoz**
Spotlight article on SIG Architecture: Conformance, the first of the series.

Closes: #436

### CODE REVIEW

**Comment by fsmunoz** [comment]
I'm still making another review pass

/hold

**Comment by fsmunoz** [comment]
/retitle: WIP: Add SIG Architecture Conformance spotlight interview

**Comment by fsmunoz** [comment]
/retitle  WIP: Add SIG Architecture Conformance spotlight interview

**Comment by Riaankl** [comment]
> /cc @Riaankl

Looking good. Thank you.

**Comment by fsmunoz** [comment]
/unhold
/retitle Add SIG Architecture Conformance spotlight interview

File: content/en/blog/2023/sig-architecture-conformance-spotlight.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+bit about yourself, your role and how you got involved in Kubernetes.
+
+**Riaan Kleinhans (RK)**: Hi! My name is Riaan Kleinhans and I live in
+South Africa. I am the Project manager for the [ii-Team](ii.nz) in New
+Zealand. When I joined ii the plan was to move to New Zealand in April
+2020 and then Covid happened. Fortunately, being a flexible and
+dynamic team we were able to make it work remotely and in very
+different time zones.
+
+The ii team have been tasked with managing the Kubernetes Conformance
```
Comment (chris-short) [inline content/en/blog/2023/sig-architecture-conformance-spotlight.md:32]: ii?

File: content/en/blog/2023/sig-architecture-conformance-spotlight.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+testing technical debt and writing tests to clear the technical
+debt. I stepped into the role of project manager to be the link
+between monitoring, test writing and the community. Through that work
+I had the privilege of meeting [Dan Kohn](https://github.com/dankohn)
```
Comment (chris-short) [inline content/en/blog/2023/sig-architecture-conformance-spotlight.md:36]: Should we link to a memorial page/post just to make sure folks understand Dan is no longer with us?

File: content/en/blog/2023/sig-architecture-conformance-spotlight.md [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+**RK**: We have reached the “100% Conformance Tested” milestone in
+release 1.27!
+
+At that point, the community took another look at all the endpoints
+that were listed as ineligible for conformance. The list was populated
+through community input over several years.  Several endpoints
+that were previously deemed ineligible for conformance have been
+identified and relocated to a new dedicated list, which is currently
+receiving focused attention for conformance test development. Again,
+that list can also be checked on apisnoop.cncf.io.
```
Comment (chris-short) [inline content/en/blog/2023/sig-architecture-conformance-spotlight.md:144]: Should we add the URL to this?

**Review by chris-short — CHANGES_REQUESTED** [review]
Good stuff! Typo and two suggestions for you to think over.

File: content/en/blog/2023/sig-architecture-conformance-spotlight.md (hunk 1 above)
Comment (Riaankl) [reply content/en/blog/2023/sig-architecture-conformance-spotlight.md:32]: We could add the url
ii.nz

File: content/en/blog/2023/sig-architecture-conformance-spotlight.md (hunk 3 above)
Comment (Riaankl) [reply content/en/blog/2023/sig-architecture-conformance-spotlight.md:144]: Here are the URL's
[dedicated list](https://github.com/kubernetes/kubernetes/blob/master/test/conformance/testdata/pending_eligible_endpoints.yaml)

[apisnoop.cncf.io](https://apisnoop.cncf.io/conformance-progress/pending-endpoints)

File: content/en/blog/2023/sig-architecture-conformance-spotlight.md
Comment (sftim) [inline content/en/blog/2023/sig-architecture-conformance-spotlight.md:1]: Hi. Can we push this back at least 7 days to find blog team review capacity?

I know that'll mean also pushing back the mirror article to match.

**Comment by sftim** [comment]
/hold
publication date not yet confirmed

File: content/en/blog/2023/sig-architecture-conformance-spotlight.md (hunk 2 above)
Comment (fsmunoz) [reply content/en/blog/2023/sig-architecture-conformance-spotlight.md:36]: Added a link to the memorial page, and also "the late" to make it clearer.

File: content/en/blog/2023/sig-architecture-conformance-spotlight.md (hunk 1 above)
Comment (fsmunoz) [reply content/en/blog/2023/sig-architecture-conformance-spotlight.md:32]: Done.

File: content/en/blog/2023/sig-architecture-conformance-spotlight.md (hunk 3 above)
Comment (fsmunoz) [reply content/en/blog/2023/sig-architecture-conformance-spotlight.md:144]: Done.

File: content/en/blog/2023/sig-architecture-conformance-spotlight.md
Comment (fsmunoz) [reply content/en/blog/2023/sig-architecture-conformance-spotlight.md:1]: Yes, will reschedule it for next Thursday, no problem.

**Comment by fsmunoz** [comment]
New date added, Oct 5 @sftim . I've reflected the suggestions up until this point  @chris-short @Riaankl , do take a quick look at the latest commit, thanks!

**Comment by sftim** [comment]
/hold cancel
/lgtm
/approve

Thanks

**Pull request merged**